android/gradletealeaf.xml

```diff
@@ -0,0 +1,9 @@
+//<!--START_PLUGINS_DEPENDENCIES-->
+	//<!--Facebook Dependencies-->
+    implementation 'com.facebook.android:facebook-core:4.34.0'
+    implementation 'com.facebook.android:facebook-login:4.34.0'
+    implementation 'com.facebook.android:facebook-places:4.34.0'
+    implementation 'com.facebook.android:facebook-messenger:4.34.0'
```

Comment:
this too. I don't think we are using it

Reply:
Actually leave it for now. We will test and remove accordingly.

Reply:
I supposed this is required by some reason. The best way is to test it by removing and trying to assemble.